po/tr.po

```diff
@@ -67,245 +67,239 @@ msgstr "Merhaba Dünya!"
+#: src/SUMMARY.md:63
 msgid "while let expressions"
-msgstr "while let İfadesi"
+msgstr "while let İfadeleri"
 
-#: src/SUMMARY.md:63
+#: src/SUMMARY.md:64
 msgid "match expressions"
-msgstr "match İfadesi"
+msgstr "match ifadeleri"
 
-#: src/SUMMARY.md:65 src/pattern-matching.md:1
+#: src/SUMMARY.md:66 src/SUMMARY.md:74 src/pattern-matching.md:1
 msgid "Pattern Matching"
-msgstr "Şablon Eşleştirme"
+msgstr "Desen Eşleştirme"
 
-#: src/SUMMARY.md:66 src/pattern-matching/destructuring-enums.md:1
+#: src/SUMMARY.md:67 src/pattern-matching/destructuring-enums.md:1
 msgid "Destructuring Enums"
-msgstr "Enumları Yok Etme"
+msgstr "Enumların Çözümlenmesi"
 
-#: src/SUMMARY.md:67 src/pattern-matching/destructuring-structs.md:1
+#: src/SUMMARY.md:68 src/pattern-matching/destructuring-structs.md:1
 msgid "Destructuring Structs"
-msgstr "Veri Yapılarını Yok Etme"
+msgstr "Yapıların (Struct) Çözümlenmesi"
 
-#: src/SUMMARY.md:68 src/pattern-matching/destructuring-arrays.md:1
+#: src/SUMMARY.md:69 src/pattern-matching/destructuring-arrays.md:1
 msgid "Destructuring Arrays"
-msgstr "Dizileri Yok Etme"
+msgstr "Dizilerin Çözümlenmesi"
 
-#: src/SUMMARY.md:69 src/pattern-matching/match-guards.md:1
+#: src/SUMMARY.md:70 src/pattern-matching/match-guards.md:1
 msgid "Match Guards"
-msgstr "Eşleştirme Muhafızları"
+msgstr "Eşleşme Koşulları/Filtreleri"
 
-#: src/SUMMARY.md:72 src/exercises/day-1/luhn.md:1
+#: src/SUMMARY.md:73 src/exercises/day-1/luhn.md:1
 #: src/exercises/day-1/solutions-afternoon.md:3
 msgid "Luhn Algorithm"
 msgstr "Luhn Algrotiması"
 
-#: src/SUMMARY.md:73
-#, fuzzy
-msgid "Pattern Matching (TBD)"
-msgstr "Şablon Eşleştirme"
-
-#: src/SUMMARY.md:75
+#: src/SUMMARY.md:76
 msgid "Day 2: Morning"
 msgstr "Gün 2: Sabah"
 
-#: src/SUMMARY.md:81 src/memory-management.md:1
+#: src/SUMMARY.md:82 src/memory-management.md:1
 msgid "Memory Management"
 msgstr "Bellek Yönetimi"
 
-#: src/SUMMARY.md:82
+#: src/SUMMARY.md:83
 msgid "Stack vs Heap"
 msgstr "Stack vs Heap"
 
-#: src/SUMMARY.md:83
+#: src/SUMMARY.md:84
 msgid "Stack Memory"
 msgstr "Stack Belleği"
 
-#: src/SUMMARY.md:84 src/memory-management/manual.md:1
+#: src/SUMMARY.md:85 src/memory-management/manual.md:1
 msgid "Manual Memory Management"
 msgstr "Manuel Bellek Yönetimi"
 
-#: src/SUMMARY.md:85 src/memory-management/scope-based.md:1
+#: src/SUMMARY.md:86 src/memory-management/scope-based.md:1
 msgid "Scope-Based Memory Management"
 msgstr "Kapsam Tabanlı Bellek Yönetimi"
 
-#: src/SUMMARY.md:86
+#: src/SUMMARY.md:87
 msgid "Garbage Collection"
 msgstr "Çöp Toplama"
 
-#: src/SUMMARY.md:87
+#: src/SUMMARY.md:88
 msgid "Rust Memory Management"
 msgstr "Rust Bellek Yönetimi"
 
-#: src/SUMMARY.md:88 src/memory-management/comparison.md:1
-msgid "Comparison"
-msgstr "Karşılaştırma"
-
 #: src/SUMMARY.md:89 src/ownership.md:1
 msgid "Ownership"
 msgstr "Sahiplik"
 
 #: src/SUMMARY.md:90 src/ownership/move-semantics.md:1
 msgid "Move Semantics"
-msgstr "Taşıma Şematiği"
+msgstr "Taşıma Semantiği"
```

Comment:
Merhaba bu [siteye](http://cayfer.bilkent.edu.tr/~cayfer/bilisim-sozlugu/tbd-ing-trk-sozluk.htm) gore Anlambilimi olarak da cevirilebilir.

Reply:
Evet, lakin çok kullanılan bir çeviri olmadığından en çok bilinen karşılığını yazdım. Konu içerisinde parantez ile "anlambilim" olarak ytazmayı düşündüm. C ve C++ ekolünden geldiğim için Türkçe eğitimlerin bir çoğu "semantik" olarak türkçeleşmiş kelimeyi kullanmakta. Buna rağmen "Anlambilimi" demeli miyiz? Yoksa konu içeriğinde parantez ile bahsetmek daha mı doğru olur sizce?